Rakefile

```diff
@@ -1,6 +1,7 @@
 require 'rubygems'
 require 'rake'
 require 'date'
+require File.expand_path('../lib/papertrail/version', __FILE__)
```

Comment:
`lib/` should be in the path when this is run so `"papertrail/version"` should be all that's necessary to require.